examples/README.md

```diff
@@ -1,3 +1,11 @@
+# Running the examples
+
+Assuming you have a checked out the puppeteer repo and run yarn (or `npm i`) to install the dependencies, the examples can be run from the root folder like so:
```

Comment:
Can you use this exact sentence (some grammatical fixes):

Assuming you have a checkout of the Puppeteer repo and have run `yarn` (or `npm i`) to install the dependencies, the examples can be run from the root folder like so:

Reply:
Done.